File: docs/training/T-H_Learning_a_new_Language/stats.py
def average(numbers):
    total = 0
    for number in numbers:
        total += number
    avg = total / len(numbers)
    return avg

def main():
    my_numbers = [3,1,6,3,5,4,6,2]
    my_avg = average(my_numbers)
    print(f"Average = {my_avg}")

if __name__ == "__main__":
    main()

File: docs/training/T-H_Learning_a_new_Language/stats.cs
public class Stats {
    public static double Average(List<int> numbers) {
        double total = 0;
        foreach (var number in numbers) {
            total += number;
        }
        var avg = total / numbers.Count;
        return avg;
    }

    public static void Main()
    {
        var my_numbers = new List<int> {3,1,6,3,5,4,6,2};
        var my_avg = Stats.Average(my_numbers);
        Console.WriteLine($"Average = {my_avg}");
    }
}